src/Client/App.fs

```diff
@@ -23,16 +27,22 @@ type Model =
     | VerbImperatives of VerbImperatives.Model
     | VerbParticiples of VerbParticiples.Model
 
+type Model = {
+    currentPage: PageModel
```

Comment:
For model members we currently have PascalCase naming, or do you think we should change it?